src/discretization.jl

```diff
@@ -119,6 +119,8 @@ discretize(torus::Torus) = discretize(torus, RegularDiscretization(50))
 
 discretize(cylsurf::CylinderSurface) = discretize(cylsurf, RegularDiscretization(50, 2))
 
+discretize(p::ParaboloidSurface) = discretize(p, RegularDiscretization(30))
```

Comment:
```suggestion
discretize(p::ParaboloidSurface) = discretize(p, RegularDiscretization(50))
```
We use 50 by default in all geometries